ops/terraform/locals.tf

```diff
@@ -12,52 +12,12 @@ locals {
       lbsubnetcidr  = "10.0.3.0/24"
     }
   }
-  dev2 = {
-    dev2 = {
-      vnetcidr      = "10.2.0.0/16"
-      appsubnetcidr = "10.2.1.0/24"
-      websubnetcidr = "10.2.2.0/24"
-      lbsubnetcidr  = "10.2.3.0/24"
-    }
-  }
-  dev3 = {
-    dev3 = {
-      vnetcidr      = "10.3.0.0/16"
-      appsubnetcidr = "10.3.1.0/24"
-      websubnetcidr = "10.3.2.0/24"
-      lbsubnetcidr  = "10.3.3.0/24"
-    }
-  }
-  dev4 = {
-    dev4 = {
-      vnetcidr      = "10.4.0.0/16"
-      appsubnetcidr = "10.4.1.0/24"
-      websubnetcidr = "10.4.2.0/24"
-      lbsubnetcidr  = "10.4.3.0/24"
-    }
-  }
-  dev5 = {
-    dev5 = {
-      vnetcidr      = "10.5.0.0/16"
-      appsubnetcidr = "10.5.1.0/24"
-      websubnetcidr = "10.5.2.0/24"
-      lbsubnetcidr  = "10.5.3.0/24"
-    }
-  }
-  dev6 = {
-    dev6 = {
-      vnetcidr      = "10.6.0.0/16"
-      appsubnetcidr = "10.6.1.0/24"
-      websubnetcidr = "10.6.2.0/24"
-      lbsubnetcidr  = "10.6.3.0/24"
-    }
-  }
   demo = {
     demo = {
-      vnetcidr      = "10.7.0.0/16"
-      appsubnetcidr = "10.7.1.0/24"
-      websubnetcidr = "10.7.2.0/24"
-      lbsubnetcidr  = "10.7.3.0/24"
+      vnetcidr      = "10.1.0.0/16"
```

Comment:
Are we changing demo to `10.1.0.0` because this wasn't used previously?  Or so we can be closer to dev's `10.0.0.0`?

Reply:
Thats true it wasn't used previously, which I guess I never thought of that lol but yes just making it closer to dev. We could make it a dramatic difference but as long as they're different to prevent overlap is the main focus here.